### PR TITLE
Add puma-plugin-systemd

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
Updated to the top hash of the branch rather than the branch itself

**Comment by ehelms** [comment]
Now based on the official 0.1.5 upstream release.

File: packages/foreman/rubygem-puma-plugin-systemd/rubygem-puma-plugin-systemd.spec [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+%description
+Puma integration with systemd: notify, status, watchdog.
+
+
+%package doc
+Summary: Documentation for %{pkg_name}
+Group: Documentation
+Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
```
Comment (evgeni) [inline packages/foreman/rubygem-puma-plugin-systemd/rubygem-puma-plugin-systemd.spec:41]: kinda duplicated?

**Review by evgeni — APPROVED** [review]

File: packages/foreman/rubygem-puma-plugin-systemd/rubygem-puma-plugin-systemd.spec (hunk 1 above)
Comment (ehelms) [reply packages/foreman/rubygem-puma-plugin-systemd/rubygem-puma-plugin-systemd.spec:41]: That comes from our template

**Pull request merged**